Bio/File.py

```diff
@@ -367,7 +385,11 @@ def __getitem__(self, key):
         return record
 
     def get(self, k, d=None):
-        """D.get(k[,d]) -> D[k] if k in D, else d.  d defaults to None."""
+        """Return the value in the dictionary.
+        
+        If key not found the second
+        attribute is returned. By default it is None.
+        """
```

Comment:
That's odd line breaking. We could explain the short hand k=key and d=default (which followed the Python built-in dictionary docstring). How about:

```
If the key (k) is not found, this returns None unless a specified
default (d) is specified.
```